docs/release-notes/flink-1.7.zh.md

```diff
@@ -108,9 +108,9 @@ Consequently, we encourage our users to enable [local recovery]({{site.baseurl}}
 Flink now properly supports `TaskManagers` with multiple slots.
 Consequently, `TaskManagers` can now be started with an arbitrary number of slots and it is no longer recommended to start them with a single slot.
 
-### StandaloneJobClusterEntrypoint generates JobGraph with fixed JobID
+### StandaloneApplicationClusterEntryPoint generates JobGraph with fixed JobID
```

Comment:
Here as well, these are the release notes.

Reply:
Same here